pkg/golang/mod/parse.go

```diff
@@ -13,12 +14,46 @@ import (
 	"github.com/aquasecurity/go-dep-parser/pkg/types"
 )
 
+var (
+	// By convention, modules with a major version equal to or above v2
+	// have it as suffix in their module path.
+	vcsUrlMajorVersionSuffixRegex = regexp.MustCompile(`(/v[\d]+)$`)
+
+	// gopkg.in/user/pkg.v -> github.com/user/pkg
+	vcsUrlGoPkgInRegexWithUser = regexp.MustCompile(`^gopkg\.in/([^/]+)/([^.]+)\..*$`)
+
+	// gopkg.in without user segment
+	// Example: gopkg.in/pkg.v3 -> github.com/go-pkg/pkg
```

Comment:
Is this rule defined somewhere?

Reply:
@knqyf263 I verified this logic from [here](https://labix.org/gopkg.in#:~:text=gopkg.in-,/,-pkg.v3%20%20%20%20%20%20%E2%86%92%20github)

Reply:
What is this website? Looks like it is not official.